backend/src/models/database.rs

```diff
@@ -0,0 +1 @@
+pub struct Db;
```

Comment:
There isn't going to be more models for this.  I think it makes most sense to make a module called `db` and in that contains the `Db` struct AND all of the queries so that imports look like this:

`use crate::db::user::{list_user, create_user}`
`use crate::db::message::{delete_message, create_messasge}`
as an example